circuits/cpp/src/aztec3/circuits/apps/opcodes/opcodes.hpp

```diff
@@ -50,10 +51,9 @@ template <typename Composer> class Opcodes {
      * - Generate constraints to prove each datum's existence in the tree
      * - Validate the data
      */
-    template <typename Note>
-    static std::vector<Note> UTXO_SLOAD(UTXOSetStateVar<Composer, Note>* utxo_set_state_var,
-                                        size_t const& num_notes,
-                                        typename Note::NotePreimage const& advice);
+    template <typename Note> static std::vector<Note> UTXO_SLOAD(UTXOSetStateVar<Composer, Note>* utxo_set_state_var,
```

Comment:
Might be more legible if we always put `template <...>` on a line before the function declaration. Not worth pauding this PR, though.

Reply:
We can choose to do have it _always_ on its own line, sometimes on its own line, or just let people use their best judgement. What do you think?